src/platform/user/authentication/utilities.js

```diff
@@ -75,12 +68,25 @@ function redirectWithGAClientId(redirectUrl) {
   }
 }
 
+export function standaloneRedirect() {
+  const searchParams = new URLSearchParams(window.location.search);
+  const application = searchParams.get('application');
+  const to = searchParams.get('to');
+  let url = externalRedirects[application] || null;
+
+  if (url && to) {
+    url = url.replace(new RegExp('/$'), '');
+    url += `/${to.replace(new RegExp('^/'), '').replace('\r\n', '')}`;
+  }
+  return url;
+}
+
 function redirect(redirectUrl, clickedEvent) {
   // Keep track of the URL to return to after auth operation.
   // If the user is coming via the standalone sign-in, redirect to the home page.
   const returnUrl =
     window.location.pathname === '/sign-in/'
-      ? window.location.origin
+      ? standaloneRedirect()
```

Comment:
```suggestion
      ? standaloneRedirect() || window.location.origin
```

Reply:
I tested above locally and it seemed to work. I think "null" isgetting written to sessionStorage and then appended to the base url in AuthApp.

Reply:
thanks for testing and fixing